### ISSUE TITLE
Taiko works different between local (mac) and GitHub actions (ubuntu runner)

### ISSUE THREAD

**Issue by mboiman** [getgauge/taiko#2241]
**Describe the bug**
I have a taiko & gauge script wich works local perfectly in headless mode with Chromium. When I let it run over github actions the step 
```
await click("name", toRightOf("QuestionName")); 
```
not working and getting:
Error Message: Error: Element matching text "nameOfTheButton" is covered by other element

```
 at node_modules/taiko/lib/actions/pageActionChecks.js:189:11
26
            at async waitAndGetActionableElement (node_modules/taiko/lib/actions/pageActionChecks.js:156:3)
27
            at async click (node_modules/taiko/lib/actions/click.js:57:21)
28
            at async module.exports.click (node_modules/taiko/lib/taiko.js:919:16)
29
            at async module.exports.<computed> (node_modules/taiko/lib/taiko.js:2622:14)
30
            at async Object.<anonymous> (tests/taiko-funktionen.js:161:2)
```

**Versions-Local:**
 - Taiko:Version: 1.2.5 (Chromium: 88.0.4314.0) RELEASE
 - OS Mac 11.5.1
 - Node.js: 16.6.1

**Github-Version**
- Taiko 1.2.6

**YML 4 GithubAction**
```
   - name: Use Node.js
        uses: actions/setup-node@v2
        with:
          node-version: '14.x'
      - name: Install dependencies
        run: |
          npm install



      - name: Set up Go 1.16
        uses: actions/setup-go@v1
        with:
          go-version: 1.16
        id: go

      - name: Build
        run: |
          go run build/make.go --verbose

      - name: Install Gauge
        run: |
          go run build/make.go --install --prefix=/tmp/gauge --verbose
          echo "/tmp/gauge/bin" >> $GITHUB_PATH

      - name: Install Taiko
        run: |
          cd gauge-taiko-test
          npm install taiko

      - name: Prep gauge-js
        run: |
          git clone https://github.com/getgauge/gauge-js.git
          cd gauge-js
          npm install
        shell: bash

      - name: Install gauge-js
        run: |
          cd gauge-js
          npm run installPlugin
          

      - name: install browser dependencies
        run: |
          cd gauge-taiko-test
          sudo apt-get update
          sudo apt-get install -y \
            libnss3 \
            libatk1.0-0 \
            libatk-bridge2.0-0 \
            libxcomposite1 \
            libcups2 \
            libxrandr2 \
            libpangocairo-1.0-0 \
            libgtk-3-0 \
            libgbm-dev 
  
```

```
gauge -v

local:
Gauge version: 1.4.0
Plugins
-------
html-report (4.0.12)
java (0.7.15)
js (2.3.17)
screenshot (0.1.0)

-------------
on the Runner:

Gauge version: 1.3.3
Commit Hash: b99fc2c

Plugins
-------
html-report (4.1.0)
java (0.7.15)
js (2.3.17)
screenshot (0.1.0)

6.14.14

```

**Additional context**
What can I do to solve this problem or to have a hint what the problem is?

**Comment by mboiman** [comment]
ok, I found it out. 
In Version 1.2.5
```
await click("name", toRightOf("QuestionName")); 
```
working better than in 1.2.6 maybe u can fix this for 1.2.7 ;)

**Comment by mboiman** [comment]
I see here is the real problem: https://github.com/getgauge/taiko/issues/2234
so I close this issue
